SwiftWisdom/Core/StandardLibrary/Array/Array+Utilities.swift

```diff
@@ -114,15 +114,30 @@ extension Array where Element: Hashable {
 }
 
 extension Array {
-    /// Provides a way to safely index into an array. If the index is beyond the array's final element this method will return `nil`.
+    /// DEPRECATED: Use `ip_safely` instead.
+    ///
+    /// Provides a way to safely index into an array. If the index is beyond the array's 
+    /// final element this method will return `nil`.
     ///
     /// - parameter safe: Index of the element to return
     ///
     /// - returns: An `Element` if the index was correct, or `nil` if it goes beyond the array
+    @available(*, deprecated: 0.8.0, message: "Use `ip_safely` instead.")
```

Comment:
Since we're going to keep this, seems like we could just bump the version to `0.7.11`